openedx/core/djangoapps/cors_csrf/admin.py

```diff
@@ -1,7 +1,11 @@
-"""Manage cross-domain configuration. """
+"""
+Manage cross-domain configuration.
+"""
+
 from django.contrib import admin
 from config_models.admin import ConfigurationModelAdmin
-from cors_csrf.models import XDomainProxyConfiguration
+
+from .models import XDomainProxyConfiguration
```

Comment:
I'm curious about best practices of using relative imports.  Of course it has the advantage of fewer changes with possible subsequent moves.  At the same time, I like being able to copy/paste an import from a test and simpler searches.  I'm not necessarily asking to change this, but I'm curious to learn what others think.